client/src/components/ratings_reviews/SideBar.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+// this component needs to render:
+//  - the Average star rating for a clicked item
+//  - the percent of reviewers that recommend this product
+//  - the breakdown of star ratings 1 -5 star ratings
+//  - how the product fits on a scale from too small to too large
+//  - how comfortable the product is from poor to perfect
+const SideBar = (props) => {
+  // console.log('this is props from sidebar:', props.metaData);
+  // need to find the average star rating
+  const starRating = props.metaData.ratings;
```

Comment:
average rating is calculated in overview too - maybe worth tossing it into store